docs/remarkable/device.md

```diff
@@ -0,0 +1,75 @@
+There are several ways to make it work, choose whatever works for you
```

Comment:
This file could be named `docs/remarkable/setup.md` or even `docs/devices/setup.md` as it describes the setup steps.

Reply:
Thanks for the suggestion, I renamed it accordingly.